src/Tribe/Views/V2/View.php

```diff
@@ -1371,4 +1391,60 @@ static function ( $value, $key ) {
 
 		return $this->should_reset_page;
 	}
+
+	/**
+	 * Returns the breadcrumbs data the View will display on the front-end.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	protected function get_breadcrumbs() {
+		$context     = $this->context;
+		$context_arr = $context->to_array();
+		$breadcrumbs = [];
+
+		// Get term slug if taxonomy is not empty
+		if ( ! empty( $context_arr[ TEC::TAXONOMY ] ) ) {
```

Comment:
Do not use the `Tribe__Context::to_array` method here.
The method is costly as it will iterate through **all** the current context locations and resolve them.
Since you just need two locations, the `taxonomy` and `tribe_events_cat` one, just get them using `Tribe__Context::get( $key, $default )`.

Reply:
will fix, thanks